src/js/app.js

```diff
@@ -9,7 +9,15 @@
    */
   ns.app = {
 
-    init : function () {
+    init: function () {
+      var localeCode = window.locale;
+      var i18n = window.locales[localeCode];
+
+      if (window.localeCode === undefined) {
```

Comment:
what's the difference between `window.locale` and `window.localeCode`?

Reply:
They are the same thing. 
 
What I did here is that is just got the value the of window.locale which is en_us for example then use it to retrieve the strings found in window.locales. You can see in the image below what window.locales represent

![image](https://user-images.githubusercontent.com/18276475/118158434-9fc52900-b3d0-11eb-8ef4-87293cbaae33.png)

But I just realized there is an error there since window.localeCode does not exist and will always be undefined! Thanks for pointing it out 

I can just simplify the code to be

```
 var i18n = window.locales[window.locale];

 if (window.locale === undefined) {
```